api/functions/promisifiedFunctions.js

```diff
@@ -0,0 +1,6 @@
+const jwt = require('jsonwebtoken');
```

Comment:
You can add more packages here that do involve functions that take a callback, and add them to the module.exports